### DISCUSSION TITLE
Creating Instance for table

### DISCUSSION THREAD

**Discussion by praajna-shettigar** [Budibase/budibase#8414] [Help]
Hi,
I have two table "Project" and "Task". with 1:N relationship
Once I create "Project" record it should save project record and create instance for "Task" table.
Relationship should match perfectly.
project field in the "Task" table should match with the "Project" created.
Help me to crack this usecase

**Comment by melohagan** [comment]
These may help: 
- https://docs.budibase.com/docs/form-steps
- https://docs.budibase.com/docs/chaining-actions#example-creating-and-linking-two-table-entries-in-one-form
